kuma/wiki/utils.py

```diff
@@ -182,3 +183,12 @@ def analytics_upageviews_by_revisions(revisions):
     start_date = min(r.created for r in revisions)
 
     return analytics_upageviews(revision_ids, start_date)
+
+
+def get_document_tags(document_id):
+    """Get the translations of the document and return locale, slug, title"""
```

Comment:
The docstring says it gets translations of the document, but I don't see it in the code. Maybe I need to check the ES return.

Reply:
Sorry, my bad.
I wrote this docstring for other function, but it got removed later. I will fix it in next commit